systemtests/src/main/java/io/enmasse/systemtest/messagingclients/MessagingClientRunner.java

```diff
@@ -5,31 +5,31 @@
 package io.enmasse.systemtest.messagingclients;
 
 import io.enmasse.api.model.MessagingEndpoint;
-import io.enmasse.api.model.MessagingEndpointPort;
 import io.enmasse.systemtest.Endpoint;
 import io.enmasse.systemtest.logs.CustomLogger;
 import io.enmasse.systemtest.messagingclients.rhea.RheaClientReceiver;
 import io.enmasse.systemtest.messagingclients.rhea.RheaClientSender;
-import io.enmasse.systemtest.messaginginfra.resources.MessagingAddressResourceType;
 import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
 
-import static io.enmasse.systemtest.messaginginfra.resources.MessagingAddressResourceType.getPort;
+import static io.enmasse.systemtest.messaginginfra.resources.MessagingEndpointResourceType.getPort;
```

Comment:
I don't like this kind of imports, it causes bad readability in code becasue nobody know from which class this static method is, so please remove this import and use `MessagingEndpointResourceType.getPort()`